airbyte-integrations/connectors/source-redshift/src/main/java/io/airbyte/integrations/source/redshift/RedshiftSource.java

```diff
@@ -51,6 +69,25 @@ private void addSsl(final List<String> additionalProperties) {
     additionalProperties.add("sslfactory=com.amazon.redshift.ssl.NonValidatingFactory");
   }
 
+  @Override
+  public List<TableInfo<CommonField<JDBCType>>> discoverInternal(JdbcDatabase database) throws Exception {
+    if (schemas != null && !schemas.isEmpty()) {
```

Comment:
is this identical to https://github.com/airbytehq/airbyte/blob/26290983900448f2442e143fb0076cafa21eb88d/airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java#L133-L147 ?

@tuliren can this (plus the `schema` config option) just be pulled up to the abstract JDBC source?

Reply:
> @tuliren can this (plus the schemas config option) just be pulled up to the abstract JDBC source?

Eventually yes. But because this is a fix for on call, I am fine with leave it as is for now, and DRY the code in a separate PR.

Reply:
I added a TODO here: https://github.com/airbytehq/airbyte/issues/2781